theme/less/_header.less

```diff
@@ -0,0 +1,69 @@
+.header-top
+{
+  padding: 25px;
+}
+
+header label select
+{
+  width: 163px;
+  color: #3A7D11;
+  font-size: 24px;
+  border: 0;
+  opacity: 0.7;
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  background: url(/images/language-arrow.png) no-repeat right;
+}
+
+header a
+{
+  text-decoration: none;
+}
+
+.header-top a
+{
+  max-width: 190px;
+}
+
+header p
+{
+  display: inline;
+  opacity: 0.7;
+  color: #000000;
```

Comment:
As we don't have too many colors as in the CBA website, thinking that we can start using `variables.less`, for colors, same applies to other places where we can make a use of them.

Reply:
You mean all colors? Foe example, we can define `@primary` as ` #3A7D11`. What variable name we can give to `#000000`? Maybe `@typographic` `@strong` `@straight`?

Reply:
I just noticed that we are using black with opacity, which make it into something very similar to the Primary color, as in general having completely black text is not a common practice, I assume in the Figma design was meant to reuse the primary color ? Anyway using opacity to adjust the color is not what we want in general.

Reply:
Meeting-notes: We will try to find a color that would fit both header and navigation items. Maybe something in between of those used currently.